packages/extension-chains/src/index.ts

```diff
@@ -40,10 +40,12 @@ export function metadataExpand (definition: MetadataDef, isPartial = false): Cha
   const isUnknown = genesisHash === '0x';
   let hasMetadata = false;
 
+  const signedExtensions = metaCalls ? new Metadata(registry, base64Decode(metaCalls)).asLatest.extrinsic.signedExtensions.toJSON() as string[] : undefined;
```

Comment:
Here we base64 decode the meta twice and create the Metadata twice. Should be done once only since it is quite a heavy operation, they are massive. (So move it inside the check below)

Reply:
thanks... made this change